src/card_draw.lua

```diff
@@ -0,0 +1,446 @@
+            if center.set == 'Default' or center.set == 'Enhanced' and not center.replace_base_card then
+                if not center.no_suit then
+                    local suit = SMODS.Suits[self.base.suit] or {}
+                    if suit.draw and type(suit.draw) == 'function' then
+                        suit:draw(self, layer)
+                    end
+                end
+                if not center.no_rank then
+                    local rank = SMODS.Ranks[self.base.value] or {}
+                    if rank.draw and type(rank.draw) == 'function' then
+                        rank:draw(self, layer)
+                    end
+                end
+            end
+        end
+    end,
+}
+SMODS.DrawStep {
+    key = 'card_type_shader',
+    order = 10,
+    func = function(self)
+        if not self.vortex and self.sprite_facing == 'front' then
+            if (self.ability.set == 'Voucher' or self.config.center.demo) and (self.ability.name ~= 'Antimatter' or not (self.config.center.discovered or self.bypass_discovery_center)) then
+                if self:should_draw_base_shader() then
+                    self.children.center:draw_shader('voucher', nil, self.ARGS.send_to_shader)
+                end
+            end
+            if (self.ability.set == 'Booster' or self.ability.set == 'Spectral') and self:should_draw_base_shader() then
+                self.children.center:draw_shader('booster', nil, self.ARGS.send_to_shader)
+            end
+        end
+    end,
+}
+
+SMODS.DrawStep {
+    key = 'edition',
+    order = 20,
+    func = function(self, layer)
+        if not self.vortex and self.sprite_facing == 'front' then
+            if self.edition then
+                for k, v in pairs(G.P_CENTER_POOLS.Edition) do
+                    if self.edition[v.key:sub(3)] and v.shader then
+                        if type(v.draw) == 'function' then
+                            v:draw(self, layer)
+                        else
+                            self.children.center:draw_shader(v.shader, nil, self.ARGS.send_to_shader)
+                            if self.children.front and self.ability.effect ~= 'Stone Card' and not self.config.center.replace_base_card then
+                                self.children.front:draw_shader(v.shader, nil, self.ARGS.send_to_shader)
+                            end
+                        end
+                    end
+                end
+            end
+            if (self.edition and self.edition.negative) or (self.ability.name == 'Antimatter' and (self.config.center.discovered or self.bypass_discovery_center)) then
+                self.children.center:draw_shader('negative_shine', nil, self.ARGS.send_to_shader)
+            end
+        end
+    end,
+}
+
+SMODS.DrawStep {
+    key = 'seal',
+    order = 30,
+    func = function(self, layer)
+        if not self.vortex and self.sprite_facing == 'front' then
+            local seal = G.P_SEALS[self.seal or {}] or {}
```

Comment:
```suggestion
            local seal = G.P_SEALS[self.seal] or {}
```

I don't believe that is nessicary

Reply:
Indeed it is not